code/ConvertToStandardPath_SubmarineCable.py

```diff
@@ -298,8 +293,10 @@ def calculate_distance_between_cable_landing_points(wktToCities, cableIdToWkt):
             start_city_name, start_city_state, start_city_country, start_city_lati, start_city_longti = start_city
             end_city_name, end_city_state, end_city_country, end_city_lati, end_city_longti = end_city
 
-            start_city_coord = (start_city_lati, start_city_longti)
-            end_city_coord = (end_city_lati, end_city_longti)
+            start_city_coord = (floatFormatter(
```

Comment:
What is the reason to limit the precision to 2 digits? Maybe explain it here too.

Reply:
a comment has been add to function floatFormatter.